.github/actions/argus-builder/manifest-update/action.yml

```diff
@@ -51,6 +51,7 @@ runs:
           }
     - uses: actions/checkout@v4
       with:
+        persist-credentials: true # creds used in Update Argus Manifest step
```

Comment:
not sure if we just accept this risk or if we want to do something about this

in my experience, automation doesn't make commits against the repo. the name of the artifact is also not something baked into the repo that get constantly updated. it's usually an external system managing the mapping of git hash to docker img tag, and that system will also keep track of what's deployed in prd/staging. sounds like that should be all done in argus so I'm curious if that's something that's possible/something we've considered doing.